README.md

```diff
@@ -187,6 +207,10 @@ python readml/main.py --help
 Usage: main.py [OPTIONS]
 
 Options:
+  --config-values    Dictionnary of configuration adapted to the type of 
```

Comment:
Same as below 
Add that templates are available in readml/config/ folder

Reply:
Done :)